_posts/2019-05-23-assessing-rails-upgrades.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Is this Rails Upgrade Project a good fit?"
+date: 2019-05-23 15:00:00
+categories: ["rails", "upgrades"]
+author: "emily"
+---
+
+Over the years we have seen a significant increase in the  demand for [Ruby on Rails](http://rubyonrails.org/) upgrades. That's why at [Ombu Labs](https://www.ombulabs.com) we decided to develop [FastRuby.io](https://fastruby.io), our Ruby on Rails upgrade service.
+
+When we receive an inquiry via [FastRuby.io](https://fastruby.io), **the first thing we like to do is schedule a call with the potential client to get more details about their app**.
+
+<!--more-->
+
+In the call we like to get the following info:
+
+1. A general overview of their app and why they are interested in upgrading Rails
+2. What **Rails version is their app currently on**?
+3. What **Rails version do they want to upgrade to**?
+4. What **Ruby version** is their app using?
+5. How large is their app? How many lines of code does it have?
```

Comment:
Usually "How many models does it have?" is a good indicator for size of the application. LoC is a tricky one...